### DISCUSSION TITLE
user-guide/plugins/about-plugin

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#76] [文档评论区]
# user-guide/plugins/about-plugin

什么是插件？插件可以做什么？ 谈到 Zotero 插件，可能指的有三种东西： 安装在 Zotero 里的，用于增强 Zotero 的功能，一般大家提起 Zotero 插件 都指的是这一类。本文档中提到的 Zotero 插件都指这一种。; （不是 Zotero 插件，是浏览器插件）安装在浏览器里的 Zotero Connecter，用于从网页上抓取文献元...

https://zotero-chinese.com/user-guide/plugins/about-plugin.html

**Comment by zzz112225** [comment]
想问下为什么我的工具栏上面找不到附加组件这一栏了，这是什么原因呢？

**Reply by northword** [reply]
如果是 Zotero 7，在工具 - Plugins。Z7 更新了这个术语，但本地化还没有跟进。

**Reply by zzz112225** [reply]
是7，找到了谢谢！！我重新加入了translate插件，但是发现没有自动划词翻译了？只能在侧窗看到，请问如何操作可以恢复自动划词翻译呢

**Reply by northword** [reply]
![image](https://github.com/zotero-chinese/wiki/assets/44738481/5f4d7cd2-d46a-499e-9c20-e23df47aa833)

如果仍未出现，尝试：重启 Zotero / 重新安装插件（确保是最新版本，可以在 zotero-chinese.com/plugins 下载新的）

**Reply by zzz112225** [reply]
谢谢！已解决！

**Comment by Maverick618** [comment]
如果出现插件安装后，在设置里没有出现该插件的选项（如果此插件应该有的话），建议重启以下Zotero。

如果还是没有，考虑是否一次性安装了多个插件，这里指的是一次性拖入多个插件的xpi安装包进入Zotero的插件管理界面；或者是在`Install Plugin From File`中同时选择多个插件的xpi安装包。如果是这样可能是插件安装出问题了，我就是遇到了这个问题。

我的办法是**将所有插件卸载，然后重新安装，一次安装只一个插件**。

**Comment by Stork343** [comment]
是不是有些插件在 Windows上能使用, 而在 MacOS 无法使用?

**Comment by Kimberly-zheng** [comment]
请问有没有什么可以画线标注影印版pdf的插件呢～

**Comment by WT0805** [comment]
help zotero7.1装不上style 5.2.3..............

**Reply by WT0805** [reply]
说错了，是装不上style5.3.5，在其他地方找了一个之前版本的style也装不上，求解答，捉急，谢谢。
7.1-beta.14+2e9b9d407 (64-bit）

**Comment by pearly7099-blip** [comment]
可以更新下zotero 10适配的插件吗？

**Reply by northword** [reply]
一直更新着的，你可以在左上角筛选器里选择10
